src/index.ts

```diff
@@ -82,6 +82,6 @@ export default {
 	};
 	SHA: SbvrType & {
 		validateSync: (value: string) => string;
-		compare: (str: string, hash: string) => Promise<boolean>;
+		compare: (str: string, hash: string) => Promise<boolean> | boolean;
```

Comment:
This is a breaking change because doing
```
sbvrTypes.SHA.compare(...).then(...)
```
would start to throw. In practice in most cases it'll probably be fine, eg `await ...` or `.then(() => sbvrTypes.SHA.compare(...))` but it's not guaranteed

Reply:
Ok, I've reverted that change. So we'll still have 1 lint warning, but I'd rather that than a breaking change in this PR.